examples/autopilot_private_firewalls/main.tf

```
@@ -33,7 +33,9 @@ provider "kubernetes" {
 }
 
 module "gke" {
-  source                            = "../../modules/beta-autopilot-private-cluster/"
+  source  = "terraform-google-modules/kubernetes-engine/google//modules/beta-autopilot-private-cluster/"
```

This might be the reason?

```suggestion
  source  = "terraform-google-modules/kubernetes-engine/google//modules/beta-autopilot-private-cluster"
```

---

Thanks, good eye

FWIW, vanilla tflint 0.53.0 with the regular ruleset doesn't complain locally.

---

@wyardley Here is the tflint config used for /example if you are curious: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/home/.tflint.example.hcl